### PR TITLE
Contrib > Remove `debounceTime`

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-contrib-remove-debounce](https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-contrib-remove-debounce)

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "editorconfig.editorconfig",
+    "ms-vscode.vscode-typescript-tslint-plugin"
+  ],
+  // List of extensions recommended by VS Code that should not be recommended for users of this workspace.
+  "unwantedRecommendations": []
+}
```
Comment (Blackbaud-AlexKingman) [inline .vscode/extensions.json:11]: Why are we committing these changes?

File: src/app/visual/autonumeric/autonumeric-visual.component.html [hunk 2]
```diff
@@ -102,11 +139,21 @@ <h2>
...
   <button
     class="sky-btn"
     type="button"
     (click)="onDisableClick()"
   >
     Toggle disabled
   </button>
+
+  <pre>{{ autonumericOptions | json }}</pre>
 </div>
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/autonumeric/autonumeric-visual.component.html:159]: When I click the 'Set Value' button in the visual demo, a console error gets thrown: 
```
ERROR Error: Must supply a value for form control with name: 'currency'.
```

File: .vscode/extensions.json (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply .vscode/extensions.json:11]: They're in our template; Thomas was doing a good deed. https://github.com/blackbaud/skyux-sdk-template/blob/master/.vscode/extensions.json

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-contrib-remove-debounce](https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-contrib-remove-debounce)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**